### ISSUE TITLE
Forecast issue

### ISSUE THREAD

**Issue by mandar-avhad** [unit8co/darts#779]
![Screenshot from 2022-02-10 00-15-40](https://user-images.githubusercontent.com/61730638/153268969-f7ced558-0e8e-4fc8-a150-6f2b3aea6a0a.png)

I getting the above forecasts.
The first is for RNN and the second is for NBEATS model.
Can anyone suggest what this issue might be?

My model training code:
For RNN

`model = RNNModel(input_chunk_length=16, output_chunk_length=12)
model.fit(train)
prediction = model.predict(800)

Series.plot(label = 'actual')
prediction.plot(label = 'forecast', lw = 3)
plt.legend()`

-------------
For NBEATS

`model = NBEATSModel(input_chunk_length=16, output_chunk_length=12)
model.fit(train)
prediction = model.predict(800)

Series.plot(label = 'actual')
prediction.plot(label = 'forecast', lw = 3)
plt.legend()`

Please suggest the best way to do this.
Also, suggest a method to do multivariate forecasting (using features other than target for forecasting). An example would be great.
Thanks!

**Comment by analyticray** [comment]
We have a similar issue. Any solution to this?

**Comment by lujaini7** [comment]
I also have similar issue regarding this matter. Please solve this issue. So headache..

**Comment by brunnedu** [comment]
Hi @mandar-avhad, when using deep learning models, it is recommended to scale the train series before fitting the model (standardize, normalize), then predict the desired forecast horizon, and after that inverse scale the predictions back to the original size. For this darts provides the [Scaler](https://unit8co.github.io/darts/generated_api/darts.dataprocessing.transformers.scaler.html?highlight=scaler#module-darts.dataprocessing.transformers.scaler) object that allows you to scale your series of any format (`TimeSeries` or `Sequence[TimeSeries]`) using any sklearn scaler such as `StandardScaler` or `MinMaxScaler`. For some examples you can also have a look at the [RNN example notebook](https://github.com/unit8co/darts/blob/master/examples/04-RNN-examples.ipynb) or the [NBeats example notebook](https://github.com/unit8co/darts/blob/master/examples/07-NBEATS-examples.ipynb).
